src/styles.scss

```diff
@@ -25,3 +25,14 @@
     @apply text-blue-600 underline;
   }
 }
+
+.nav-item {
+  &.active {
+    background-color: lavender;
+  }
+
+  &:hover {
+    cursor: pointer;
+    background-color: aliceblue;
+  }
+}
```

Comment:
Nah. Styles of a specific component should be right where that component itself is, not global.

Also if opted to use tailwind, use tailwind, do not do manual styles.